spyder/app/mainwindow.py

```diff
@@ -1446,17 +1446,27 @@ def show_DPI_change_message(self):
         screen = self.window().windowHandle().screen()
         screen.logicalDotsPerInchChanged.disconnect(
             self.show_DPI_change_message)
-        QMessageBox.warning(
+        answer = QMessageBox.warning(
             self, _("Warning"),
             _("A monitor scale change was detected. <br><br>"
-              "Spyder will be restarted to be properly displayed."),
-            QMessageBox.Ok)
-        # Activate HDPI auto-scaling option since is needed for a proper
-        # display when using OS scaling
-        CONF.set('main', 'normal_screen_resolution', False)
-        CONF.set('main', 'high_dpi_scaling', True)
-        CONF.set('main', 'high_dpi_custom_scale_factor', False)
-        self.restart()
+              "We recommend restarting Spyder to ensure a "
+              "proper display. <br><br>"
+              "Do you want to restart Spyder?"),

```

Comment:
Let's change this to

```
We recommend restarting Spyder to ensure that it's properly displayed.
Do you want to restart it?
```

Reply:
Ok, let's add the following to make sure that users set the right option too:

```
We recommend restarting Spyder to ensure that it's properly displayed.<br>
If you don't want to do that, please be sure to activate the option<br><br>
<tt>Enable auto high DPI scaling</tt><br><br>
in <tt>Preferences > General > Interface</tt>, in case Spyder is not displayed
correctly.<br><br>
Do you want to restart Spyder?
```

Reply:
@ccordoba12 a preview:
![image](https://user-images.githubusercontent.com/16781833/72819345-161da500-3c3b-11ea-8adc-d523e4a14755.png)

Reply:
Thanks @dalthviz! I did a minor change (my fault), but the rest looks good!